impeller/playground/playground_test.cc

```diff
@@ -22,6 +24,8 @@ void PlaygroundTest::SetUp() {
   }
 
   SetupWindow(GetParam());
+
+  start_time_ = fml::TimePoint::Now().ToEpochDelta().ToSecondsF();
```

Comment:
If you retain `start_time_` as a `TimeDelta` (that is skip the `.ToSecondsF()` here), then `GetSecondsElapsed()` becomes `(fml::TimePoint::Now().ToEpochDelta() - start_time_).ToSecondsF()`, and you only have one int -> double conversion instead of two.

Reply:
Nice, done.